Formula/go-boring.rb

```diff
@@ -0,0 +1,74 @@
+class GoBoring < Formula
+  desc "Go programming language with BoringCrypto"
+  homepage "https://go.googlesource.com/go/+/dev.boringcrypto/README.boringcrypto.md"
+  url "https://go-boringcrypto.storage.googleapis.com/go1.16.2b7.src.tar.gz"
+  sha256 "8991c28e30bdd9b34076c2871ab69aac7f306dba08d202aa7301f3adb54ad9f3"
+  license "BSD-3-Clause"
+
+  keg_only "go alternative"
```

Comment:
```suggestion
  keg_only "it conflicts with the Go formula"
```
Also, you need a `livecheck` block. https://docs.brew.sh/Brew-Livecheck

Reply:
added livecheck block